other/disallow-proc-self-fd-workingdir-cve-2024-21626/disallow-proc-self-fd-workingdir-cve-2024-21626.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: disallow-proc-self-fd-workingdir-cve-2024-21626
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
+    - name: no-proc-self-fd-images
+      match:
+        any:
+        - resources:
+            kinds:
+              - Pod
+      validate:
+        message: "Using /proc/self/fd in workingDir is not allowed CVE-2024-21626."
+        pattern:
+          spec:
+            containers:
+              - =(workingDir): "!/proc/self/fd*"
+            initContainers:
+              - =(workingDir): "!/proc/self/fd*"
+            ephemeralContainers:
+              - =(workingDir): "!/proc/self/fd*"
```

Comment:
Needs to also account for fixes raised in [this Slack thread](https://cloud-native.slack.com/archives/C02CEARLA3D/p1718112698475069).